### ISSUE TITLE
Random crashes in com.google.GoogleConfigService.FIRRemoteConfig

### ISSUE THREAD

**Issue by leandromperez** [firebase/firebase-ios-sdk#3602]
Hi guys, we're seeing crashes in Fabric related to the activation of the Firebase Remote Config. I'm opening this issue in case anyone else is experiencing something similar.

  * **Xcode version**: Version 10.2.1 (10E1001)
  * **Firebase SDK version**: 
**- Firebase/Core (6.5.0):**
    - Firebase/CoreOnly
    - FirebaseAnalytics (= 6.0.4)
  - Firebase/CoreOnly (6.5.0):
    - FirebaseCore (= 6.1.0)
  **- Firebase/Performance (6.5.0):**
    - Firebase/CoreOnly
    - FirebasePerformance (~> 3.1.1)
  - Firebase/RemoteConfig (6.5.0):
    - Firebase/CoreOnly
    - FirebaseRemoteConfig (~> 4.2.1)
  - FirebaseABTesting (3.0.0):
    - FirebaseCore (~> 6.0)
    - Protobuf (~> 3.5)
  - FirebaseAnalytics (6.0.4):
    - FirebaseCore (~> 6.1)
    - FirebaseInstanceID (~> 4.2)
    - GoogleAppMeasurement (= 6.0.4)
    - GoogleUtilities/AppDelegateSwizzler (~> 6.0)
    - GoogleUtilities/MethodSwizzler (~> 6.0)
    - GoogleUtilities/Network (~> 6.0)
    - "GoogleUtilities/NSData+zlib (~> 6.0)"
    - nanopb (~> 0.3)
  - FirebaseCore (6.1.0):
    - GoogleUtilities/Environment (~> 6.0)
    - GoogleUtilities/Logger (~> 6.0)
  - FirebaseInstanceID (4.2.2):
    - FirebaseCore (~> 6.0)
    - GoogleUtilities/Environment (~> 6.0)
    - GoogleUtilities/UserDefaults (~> 6.0)
  - FirebasePerformance (3.1.1):
    - FirebaseCore (~> 6.0)
    - FirebaseInstanceID (~> 4.2)
    - FirebaseRemoteConfig (~> 4.2)
    - GoogleToolboxForMac/Logger (~> 2.1)
    - "GoogleToolboxForMac/NSData+zlib (~> 2.1)"
    - GoogleUtilities/Environment (~> 6.0)
    - GoogleUtilities/ISASwizzler (~> 6.0)
    - GoogleUtilities/MethodSwizzler (~> 6.0)
    - GTMSessionFetcher/Core (~> 1.1)
    - Protobuf (~> 3.5)
  - FirebaseRemoteConfig (4.2.1):
    - FirebaseABTesting (~> 3.0)
    - FirebaseCore (~> 6.1)
    - FirebaseInstanceID (~> 4.2)
    - GoogleUtilities/Environment (~> 6.0)
    - "GoogleUtilities/NSData+zlib (~> 6.0)"
    - Protobuf (~> 3.5)

  * **Firebase Component**: RemoteConfig
  * **Component version**: 6.5.0

### [REQUIRED] Step 3: Describe the problem
We are getting random crashes when activating the remote config.

#### Steps to reproduce:
We don't have steps, we're just seeing random crashes in Fabric with this stack trace:

#### Relevant Code:

```
Crashed: com.google.GoogleConfigService.FIRRemoteConfig
0  libobjc.A.dylib                0x19b773530 objc_msgSend + 16
1  CoreFoundation                 0x19c47135c -[__NSDictionaryM objectForKeyedSubscript:] + 216
2  OurApp                           0x102d2dc1c -[RCNConfigContent copyFromDictionary:toSource:forNamespace:] + 4370422812
3  OurApp                           0x102d2a080 __49-[FIRRemoteConfig activateWithCompletionHandler:]_block_invoke + 4370407552
4  libdispatch.dylib              0x19bfc0a38 _dispatch_call_block_and_release + 24
5  libdispatch.dylib              0x19bfc17d4 _dispatch_client_callout + 16
6  libdispatch.dylib              0x19bf6a324 _dispatch_lane_serial_drain$VARIANT$mp + 592
7  libdispatch.dylib              0x19bf6ae40 _dispatch_lane_invoke$VARIANT$mp + 428
8  libdispatch.dylib              0x19bf734ac _dispatch_workloop_worker_thread + 596
9  libsystem_pthread.dylib        0x19c1a2114 _pthread_wqthread + 304
10 libsystem_pthread.dylib        0x19c1a4cd4 start_wqthread + 4
```

**Comment by morganchen12** [comment]
This looks very similar to a series of crashes that were fixed but have not yet been released. The fixes are scheduled to be released soon; can you test when 6.7.0 is out?

**Comment by leandromperez** [comment]
I will, and I'll keep you posted @morganchen12

**Comment by paulb777** [comment]
Going to close, since the issue is expected to be fixed in the upcoming 6.7.0 release.
